node/package-lock.json

```diff
@@ -1,8 +1,1391 @@
 {
   "name": "azure-pipelines-task-lib",
-  "version": "4.0.0-preview",
-  "lockfileVersion": 1,
+  "version": "4.0.0-preview.0",
+  "lockfileVersion": 2,
```

Comment:
Could you please clarify why we have to change the `lockfileVersion` version?

Reply:
Because version 2 is the standard version for npm that comes with Node 16